src/commands/graph/graph-init.js

```diff
@@ -29,30 +30,11 @@ const graphInit = async (options, command) => {
     )
   }
 
-  let [netlifyToken, loginLocation] = await getToken()
+  let [netlifyToken] = await getToken()
   if (!netlifyToken) {
     netlifyToken = await command.authenticate()
   }
 
-  let siteData = null
-  try {
-    // @ts-ignore: we need better types for our api object
-    siteData = await api.getSite({ siteId })
```

Comment:
I deleted this block of code because everything we need should already be available in `command.netlify.siteInfo` -- is this an incorrect assumption on my part?